src/logic/actions/DisplayActions.ts

```diff
@@ -0,0 +1,62 @@
+import {EditorModel} from "../../staticModels/EditorModel";
+import {NumberUtil} from "../../utils/NumberUtil";
+import {DisplaySettings} from "../../settings/DisplaySettings";
+import {ISize} from "../../interfaces/ISize";
+import {IRect} from "../../interfaces/IRect";
+import {ImageUtil} from "../../utils/ImageUtil";
+import {RectUtil} from "../../utils/RectUtil";
+import {IPoint} from "../../interfaces/IPoint";
+import {PointUtil} from "../../utils/PointUtil";
+
+export class DisplayActions {
```

Comment:
może ViewportActions? bo na takie akcje mi wyglądają metody zawarte w tym pliku :P

Reply:
Szukałem nazwy. Pasuje mi to. Tu mają być wszystkie rzeczy , które nie wiem nie są powiązane z rysowaniem na canvas.